version_test.go

```diff
@@ -704,10 +704,12 @@ func TestIncPreRelease(t *testing.T) {
 		{"v21.2.0-alpha.1", "v21.2.0-alpha.2", false},
 		{"v21.1.0-beta.3", "v21.1.0-beta.4", false},
 		{"v21.1.0-rc.3", "v21.1.0-rc.4", false},
+		{"v21.1.0-rc.3-cloudonly.1", "", true},
 		{"v21.1.0-cloudonly.1", "", true},
 		{"v21.1.0", "", true},
 		{"v21.1.8", "", true},
 		{"v26.4.8", "", true},
+		{"v26.4.8-cloudonly.1", "", true},
```

Comment:
Just confirming my understanding -- this case returns an error because `-cloudonly.1` is not a pre-release, right?

Reply:
Correct.